base64ct/src/lib.rs

```diff
@@ -202,12 +178,22 @@ pub fn decode<'a>(src: &str, dst: &'a mut [u8], padded: bool) -> Result<&'a [u8]
 }
 
 /// Decode unpadded Base64-encoded string in-place.
-// TODO(tarcieri): support for padded Base64
-pub fn decode_in_place_unpadded(buf: &mut [u8]) -> Result<&[u8], InvalidEncodingError> {
+pub fn decode_in_place(buf: &mut [u8], padded: bool) -> Result<&[u8], InvalidEncodingError> {
```

Comment:
It could be worth to mark this function `#[inline]`, to make it easier for compiler propagation of a constant `padded`. Same for other functions which use `padded`.

Reply:
Done